common/utils/src/main/resources/error/error-classes.json

```diff
@@ -6705,6 +6705,259 @@
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
   },
+  "_LEGACY_ERROR_TEMP_3050" : {
```

Comment:
So, every errors are registered under `_LEGACY`?

Reply:
Yep, I just added legacy error class with the same error messages. So later, we need to improve the error messages, write tests if there are no + update docs as we are doing in sub-tasks of https://issues.apache.org/jira/browse/SPARK-37935